App/PHP_HTML/home.php

```diff
@@ -88,6 +88,13 @@
 </div>
 </section>
 
+<script>
```

Comment:
Would it be possible to have this repeated script inside of a script file that is directly called instead of repeating the script?

Reply:
Ok